packages/e2e-tests/config/setup-debug-artifacts.js

```diff
@@ -52,3 +60,61 @@ function slugify( testName ) {
 		.replace( / /g, '-' );
 	return slug;
 }
+
+let frames = [];
+let session;
+
+const startVideo = async () => {
+	frames = [];
+	if ( ! session ) {
+		session = await page.target().createCDPSession();
+	}
+	await session.send( 'Page.startScreencast', {
+		format: 'png',
+		everyNthFrame: 1,
+	} );
+	session.on( 'Page.screencastFrame', ( event ) => {
```

Comment:
do we need to register a new listener for every started test? do they get cleared on `stopScreencast`?